packages/smooth_app/lib/pages/user_management/login_page.dart

```diff
@@ -344,33 +344,78 @@ class _LoginPageState extends State<LoginPage> with TraceableClientMixin {
 
   Future<void> showInAppReviewIfNecessary(BuildContext context) async {
     final UserPreferences preferences = context.read<UserPreferences>();
-
     if (!preferences.inAppReviewAlreadyAsked) {
       assert(mounted);
+      final bool? enjoyingApp = await showDialog<bool>(
+        context: context,
+        builder: (BuildContext context) {
+          final AppLocalizations appLocalizations =
+              AppLocalizations.of(context);
 
-      if (await showDialog<bool>(
-            context: context,
-            builder: (BuildContext context) {
-              final AppLocalizations appLocalizations =
-                  AppLocalizations.of(context);
+          return SmoothAlertDialog(
+            body: Text(appLocalizations.app_rating_dialog_title_enjoying_app),
+            positiveAction: SmoothActionButton(
+              text: appLocalizations
+                  .app_rating_dialog_title_enjoying_positive_actions,
+              onPressed: () async => Navigator.of(context).pop(true),
```

Comment:
```suggestion
              onPressed: () => Navigator.of(context).pop(true),
```
This doesn't need to be async